nestkernel/send_buffer_position.h

```diff
@@ -54,11 +54,32 @@ class SendBufferPosition
   SendBufferPosition( const AssignedRanks& assigned_ranks,
     const unsigned int send_recv_count_per_rank );
 
+  /**
+   * Returns current index of specified rank in MPI buffer.
+   */
   unsigned int idx( const thread rank ) const;
+
+  /**
+   * Returns begin index of specified rank in MPI buffer.
+   */
   unsigned int begin( const thread rank ) const;
+
+  /**
+   * Returns end index of specified rank in MPI buffer.
+   */
   unsigned int end( const thread rank ) const;
 
+  /**
+   * Returns whether the part of the buffer on the specified rank has been
+   * filled.
+   *
+   * @param rank Rank denoting which part of the buffer we check
+   */
   bool is_chunk_filled( const thread rank ) const;
+
+  /**
+   * Returns whether the entire buffer has been filled.
```

Comment:
this actually just checks whether all parts of the MPI buffer assigned to this thread are filled, not the whole MPI buffer

Reply:
Fixed!